libs/payments/ui/src/lib/client/components/CheckoutForm.tsx

```diff
@@ -135,71 +146,102 @@ export function CheckoutForm({ readOnly, cart, locale }: CheckoutFormProps) {
-              type="text"
-              data-testid="name"
-              placeholder={l10n.getString(
-                'payment-name-placeholder',
-                {},
-                'Full Name'
-              )}
-              readOnly={readOnly}
-              value={fullName}
-              onChange={(e) => {
-                setFullName(e.target.value);
-                setHasFullNameError(!e.target.value);
-              }}
-            />
-          </Form.Control>
-          {hasFullNameError && (
-            <Form.Message asChild>
-              <p className="text-sm mt-1 text-alert-red">
-                Please enter your name
-              </p>
-            </Form.Message>
-          )}
-        </Form.Field>
-      )}
-      <PaymentElement
-        options={{
-          layout: {
-            type: 'accordion',
-            defaultCollapsed: false,
-            radios: false,
-            spacedAccordionItems: true,
-          },
-          readOnly,
+    <Form.Root onSubmit={submitHandler}>
+      <CheckoutCheckbox
+        isRequired={showConsentError}
+        termsOfService={cmsCommonContent.termsOfServiceUrl}
+        privacyNotice={cmsCommonContent.privacyNoticeUrl}
+        notifyCheckboxChange={(consentCheckbox) => {
+          setFormEnabled(consentCheckbox);
+          setShowConsentError(true);
         }}
       />
-      {!isPaymentElementLoading && (
-        <Form.Submit asChild>
-          <Localized id="next-new-user-submit">
-            <PrimaryButton
-              type="submit"
-              aria-disabled={
-                !stripeFieldsComplete || !nonStripeFieldsComplete || loading
-              }
-            >
-              <Image
-                src={LockImage}
-                className="h-4 w-4 my-0 mx-3 relative top-0.5"
-                alt=""
+      <div
+        className={
+          formEnabled
+            ? ''
+            : 'relative cursor-not-allowed focus:border-blue-400 focus:outline-none focus:shadow-input-blue-focus after:absolute after:content-[""] after:top-0 after:left-0 after:w-full after:h-full after:bg-white after:opacity-50 after:z-10'
+        }
+        onClick={() => setShowConsentError(true)}
+      >
+        <Localized id="next-new-user-card-title">
+          <h3 className="font-semibold text-grey-600 text-start">
+            Enter your card information
+          </h3>
+        </Localized>
+        {!isPaymentElementLoading && (
+          <Form.Field
+            name="name"
+            serverInvalid={hasFullNameError}
+            className="my-6"
+          >
+            <Form.Label className="text-[15px] text-grey-400 block mb-1 text-start">
```

Comment:
Would built-in `text-sm` (14px) or `text-base` (16px) work instead of a [custom font size](https://tailwindcss.com/docs/font-size)?

Reply:
I updated this to use a custom size for a more unified look with the labels and error messages used in the Stripe Payment Element (15px). 

Note: the base font size for the Stripe Payment Element is currently set to 16px.

Reply:
Fly-by comment: 

I guess there's no way to update the stripe payment element? Sad. Can we at least use `rem`s, or is stripe payment also `px`?

Reply:
Apologies, clarification on the note - we had set the Stripe Payment Element to have a base font-size of 16px. 

It looks like the Element does its own calculations from the base font size, so that its labels and error message use 15px (.93rem).

Reply:
Looking at the Stripe Appearance API, it might be possible to set the Stripe Elements sizes. https://docs.stripe.com/elements/appearance-api